flake.nix

```diff
@@ -12,9 +12,10 @@
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+    nixpkgs-risc0-rustc.url = "github:NixOS/nixpkgs/34d8dbb93ddf91fb665b186d1c832b2d2f8e7ff7";
```

Comment:
Maybe add a comment on top,
Latest nixpkgs revision including the cached rustc version that risc0's rust is based on

Reply:
added